modules/lotamePanoramaIdSystem.js

```diff
@@ -252,6 +253,13 @@ export const lotamePanoramaIdSubmodule = {
       usPrivacy = getFromStorage('us_privacy');
     }
 
+    const getRequestHost = function() {
+      if (navigator.userAgent && navigator.userAgent.indexOf('Safari') != -1 && navigator.userAgent.indexOf('Chrome') == -1) {
```

Comment:
what about this https://chromium.googlesource.com/chromium/src/+/refs/heads/main/docs/ios/user_agent.md

Reply:
Thanks for calling that out, but the reality is that any ITP browsers should be using the cookieless domain. This includes Chrome and Firefox on iOS. The user agent detection as written will cover both cases.